lib/sql_tagger/pg.rb

```diff
@@ -0,0 +1,32 @@
+require 'sql_tagger'
+require 'pg'
+
+# Adapter for +PG::Connection+ from the pg gem
+module SqlTagger::PG
+  extend SqlTagger::ModuleMethods
+
+  # I wouldn't be surprised if I missed a method
```

Comment:
Looking through the docs I don't see any discrepancies, however can you modify this comment to indicate the version of pg gem API used?